stockfish/models.py

```diff
@@ -534,7 +537,9 @@ def get_top_moves(self, num_top_moves: int = 5) -> List[dict]:
         if num_top_moves != self._parameters["MultiPV"]:
             self._set_option("MultiPV", num_top_moves)
             self._parameters.update({"MultiPV": num_top_moves})
-        self._go()
+
+        self._go() if not num_nodes > 0 else self._go_nodes(num_nodes)
```

Comment:
A bug, see https://github.com/knutole/stockfish/commit/5bdd800a6182fd2a7b205c5ead4e2e3cb70831d6
```suggestion
        self._go() if num_nodes == 0 else self._go_nodes(num_nodes)
```